### ISSUE TITLE
1.0.0 got no method error for IE

### ISSUE THREAD

**Issue by PikachuEXE** [fnando/browser#171]
# Backtrace
```
NoMethodError: undefined method `split' for nil:NilClass
browser-1.0.0/lib/browser/methods/ie.rb:30→ msie_version
browser-1.0.0/lib/browser/methods/ie.rb:18→ ie_version
browser-1.0.0/lib/browser.rb:128→ version
browser-1.0.0/lib/browser/methods/ie.rb:60→ ie9?
```

# User Agent Strings
- `Mozilla/5.0 (compatible; MSIE or Firefox mutant; not on Windows server;) Daumoa 4.0`
- `Mozilla/4.0 (compatible; MSIE8.0; Windows NT 6.0) .NET CLR 2.0.50727)`

**Comment by fnando** [comment]
Thanks! What browsers generate these UAs?

**Comment by PikachuEXE** [comment]
I have no idea
I can only read the error reports

**Comment by fnando** [comment]
Found this: http://tab.search.daum.net/aboutWebSearch_en.html

**Comment by fnando** [comment]
Just released v1.0.1. Let me know of other issues related to bots that identify themselves as IE.

**Comment by PikachuEXE** [comment]
Thx!
